drivers/gles3/shaders/canvas.glsl

```diff
@@ -285,7 +285,21 @@ MATERIAL_UNIFORMS
 
 FRAGMENT_SHADER_GLOBALS
 
-void light_compute(inout vec4 light,vec2 light_vec,float light_height,vec4 light_color,vec2 light_uv,vec4 shadow,vec3 normal,vec2 uv,vec2 screen_uv,vec4 color) {
+void light_compute(
+	inout vec4 light,
+	vec2 light_vec,
+	float light_height,
+	vec4 light_color,
+	vec2 light_uv,
+#if defined(SHADOW_COLOR_USED)
```

Comment:
I think if you pass a variable that was never used, the shader compiler will optimize it out, so this is probably unnecesary, just put the variable somewhere and dont use it